projects/kit/components/progress/progress-bar/progress-bar.component.less

```diff
@@ -0,0 +1,72 @@
+@import 'taiga-ui-local';
+
+.clearProgress() {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
+}
+
+/* Used to style outer progress bar (container) */
+.progressContainer(@property, @value) {
+    @{property}: @value; // IE | Edge | Mozilla
+
+    &::-webkit-progress-bar {
+        @{property}: @value; // Chrome | Opera | Safari
+    }
+}
+
+/* used to style inner value bar */
+.progressValue(@property, @value) {
+    @{property}: @value; // Edge
+
+    &::-webkit-progress-value {
+        @{property}: @value; // Chrome | Opera | Safari
+    }
+
+    &::-moz-progress-bar {
+        @{property}: @value; // Mozilla
+    }
+}
+
+/*
+* Edge and IE animate the progress bar right out of the box
+* Chrome | Opera | Safari animation is controlled by its mixins
+* No possibility for firefox animation (https://snook.ca/archives/html_and_css/animating-progress)
+*/
+.progressAnimation() {
+    &::-webkit-progress-value {
+        transition: width var(--tui-duration) linear;
+    }
+}
+
+.progressValueColor(@color) {
+    color: @color; // IE | Not Chromium Edge
+
+    &::-webkit-progress-value {
+        background: @color; // Chromium Edge | Chrome | Opera | Safari
+    }
+
+    &::-moz-progress-bar {
+        background: @color; // Mozilla
+    }
+}
+
+:host {
+    .clearProgress();
+    .progressAnimation();
+    .progressValueColor(var(--tui-progress-color));
+    .progressContainer(background-color, var(--tui-base-03));
+
+    border-radius: var(--tui-radius-xs);
+    overflow: hidden;
+
+    &[data-size='m'] {
+        height: 1.5rem;
+    }
+
+    &[data-size='s'] {
+        height: 0.5rem;
+        .progressValue(border-radius, var(--tui-radius-xs));
```

Comment:
mixins go first. I believe there's a lint rule for that, at least in proprietary section

Reply:
fixed